Maple2.File.Parser/Xml/Table/PremiumClubPackage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+using M2dXmlGenerator;
+using Maple2.File.Parser.Enum;
+
+namespace Maple2.File.Parser.Xml.Table;
+
+// ./data/xml/table/vipgoodstable.xml
+[XmlRoot("ms2")]
+public class PremiumClubPackageRoot {
+    [XmlElement] public List<PremiumClubPackage> goods;
+}
+
+public partial class PremiumClubPackage : IFeatureLocale {
+    [XmlAttribute] public int id;
+    [XmlAttribute] public bool disable;
+    [XmlAttribute] public string salesStartDate;
+    [XmlAttribute] public string salesEndDate;
+    [XmlAttribute] public int vipPeriod;
```

Comment:
![image](https://user-images.githubusercontent.com/7710340/200246258-b4e2e664-af05-43c0-a548-d595c5b503c1.png)

`vipPeriod` is `long`, `price` is `long`

missing: `displayOrder`, `salePrice`, `saleTag`, `buyLimit`, `buyLimitResetType`